app/controllers/items_controller.rb

```diff
@@ -1,60 +1,11 @@
 class ItemsController<ApplicationController
 
   def index
-    if params[:merchant_id]
-      @merchant = Merchant.find(params[:merchant_id])
-      @items = @merchant.items
-    else
-      @items = Item.active_items
-    end
+    @items = Item.all
   end
 
   def show
     @item = Item.find(params[:id])
   end
 
-  def new
```

Comment:
all these were duplicate actions, merchants and admins are the only users who can create, edit and destroy items so there was no need for all these extra actions to be here